Recipes/Along-slope-velocities.ipynb

```diff
@@ -23,6 +23,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    u, v = u_intake_dict.get('ocean.3mon'), v_intake_dict.get('ocean.1mon')
</pre><p>Would be good to slow down at this point and play with some 'chunking' - I don't know how big these data are, but the runtime is suggesting to me quite large...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>